examples/robot_test_suite/test_uds/demo_uds.robot

```diff
@@ -12,3 +12,17 @@ Go in default session
     ${response} =  Send uds raw    \x10\x01    uds_aux
 
     Log  ${response}
+
+Use tester present sender
+    [Documentation]    If no communication is exchanged with the client for more than 5
+    ...                seconds the control unit automatically exits the current session and
+    ...                returns to the "Default Session" back, and might go to sleep mode.
+    ...
+    ...                To avoid this issue if test steps take too long between uds commands,
+    ...                the tester present sender can be use. It will send
+    ...                at a define period a Tester Present, to signal to the device that
+    ...                the client is still present.
+
+    Start tester present sender    1    uds_aux
```

Comment:
```suggestion
    Start tester present with   1 second   uds_aux
```
?

Reply:
Hum... maybe I don't get the robot syntax, what I wrote is not correct?

Reply:
No, it should be more intuitive, when it comes to the robot syntax. Therefor, you should make it more clear.

When I read your syntax above, I understand that "I start a tester present sender number 1 with? uds_aux" So there are many different one?

Reply:
Done